agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/configuration/Configuration.java

```diff
@@ -21,7 +21,72 @@
 
 package com.microsoft.applicationinsights.agent.bootstrap.configuration;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.squareup.moshi.Json;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+
 public class Configuration {
 
-    public InstrumentationSettings instrumentationSettings = new InstrumentationSettings();
+    public String connectionString;
+    public Role role = new Role();
+    public Map<String, String> customDimensions = new HashMap<>();
+    public Sampling sampling = new Sampling();
+    public List<JmxMetric> jmxMetrics = new ArrayList<>();
+    public Map<String, Map<String, Object>> instrumentation = new HashMap<String, Map<String, Object>>();
+    public Heartbeat heartbeat = new Heartbeat();
+    public Proxy proxy = new Proxy();
+    public PreviewConfiguration preview = new PreviewConfiguration();
+
+    public static class Role {
+
+        public String name;
+        public String instance;
+    }
+
+    public static class Sampling {
+
```

Comment:
do you use a formatter for this?  are you intentionally leave a new line right after the curly brace?  when ever i see this, i tempt to delete it 🤣